include/multipass/utils.h

```diff
@@ -72,6 +72,7 @@ std::string run_cmd_for_output(const QString& cmd, const QStringList& args, cons
 std::string& trim_end(std::string& s);
 std::string& trim_newline(std::string& s);
 std::string escape_char(const std::string& s, char c);
+std::string escape_all(const std::string& s);
```

Comment:
Should this be more explicit about what it does? `escape_for_shell` or some such?